### PR TITLE
Added solution to the monad exercise F(X) = R -> X

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: tex/solutions.tex [hunk 1]
```diff
@@ -688,7 +688,46 @@ \section{Solutions}
 \end{enumerate}
 \end{solution}
 
+\begin{solution}[\cref{exer:kleisli_triple_familiesofelements}]\label{sol:kleisli_triple_familiesofelements}
+For each set $X\in\Ob \SET$, we define:
+\[
+\eta_X : X \to (100 X) : x\mapsto (\lambda ?, x).
```
Comment (benediktahrens) [inline tex/solutions.tex]: Why does it say "100" here?

Comment (benediktahrens) [reply tex/solutions.tex]: I think instead of using a "?" to indicate a variable, and underscore would be more suitable (typed as `\_`).

**Review by benediktahrens — APPROVED** [review]
Looks great modulo small comments, thanks a lot!

File: tex/solutions.tex (hunk 1 above)
Comment (ghost) [reply tex/solutions.tex]: > Why does it say "100" here?
That is a very good question, I have no idea.

**Pull request merged**